src/main/java/net/snowflake/ingest/streaming/internal/BlobBuilder.java

```diff
@@ -140,7 +141,7 @@ static <T> Blob constructBlobAndMetadata(
 
         if (internalParameterProvider.setIcebergSpecificFieldsInEp()) {
           chunkMetadataBuilder
-              .setMajorVersion(Constants.PARQUET_MAJOR_VERSION)
+              .setMajorVersion(ParquetFileWriter.CURRENT_VERSION)
```

Comment:
I'd much rather depend on our own constant than a third-party library's constant, I thought I had left a comment on this but don't see it anywhere :(

Reply:
ok to take in next PR too, just remove the import whenever you revert this.